heat/core/stride_tricks.py

```diff
@@ -90,6 +90,10 @@ def sanitize_axis(shape, axis):
     >>> sanitize_axis((5, 4), 1.0)
     TypeError
     """
+    # scalars are handled like unsplit matrices
+    if len(shape) == 0:
```

Comment:
can you change this to use `numel()`? this would allow for the case that there is a 1 element torch tensor in a DNDarray

Reply:
So the split axis of 1x1 matrices shall also be None?

Reply:
So the split axis change shall be expended to higher dimensional tensors like this `np.prod(shape) == 1`? This change will let some unit tests fail.

Reply:
Scalars should be zero-dimensional